test/lib/job.test.js

```diff
@@ -442,6 +442,17 @@ describe('Job Model', () => {
                 assert.calledOnce(datastore.update);
             });
         });
+
+        it('remove periodic job', () => {
+            job.state = 'DISABLED';
```

Comment:
Is this needed? Job's `ENABLED/DISABLED` status seems to be unrelated.

Reply:
fixed